scalafix-cli/src/main/scala/scalafix/cli/Cli.scala

```diff
@@ -289,36 +299,28 @@ object Cli {
     cause.printStackTrace(options.common.err)
   }
 
-  def parsed(code: Input, config: ScalafixConfig): Either[Failed, Tree] = {
-    config.parser.apply(code, config.dialect) match {
-      case Parsed.Success(ast) => Right(ast)
-      case Parsed.Error(pos, msg, e) =>
-        Left(Fixed.Failed(Failure.ParseError(pos, msg, e)))
-    }
-  }
-
   def handleFile(file: File, options: ScalafixOptions): ExitStatus = {
     val config =
       if (file.getAbsolutePath.endsWith(".sbt")) options.resolvedSbtConfig
       else options.resolvedConfig
-
-    val fixed = Scalafix.fix(Input.File(file), config.get)
+    val fixed =
+      Try(options.resolvedRewrite.get.apply(Input.File(file), config.get))
```

Comment:
It's just sloppy coding, been meaning to refactor the cli a bit to get rid of those